templates/_helpers.tpl

```diff
@@ -0,0 +1,60 @@
+{{- define "onetimesecret-chart.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "onetimesecret-chart.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "onetimesecret-chart.labels" -}}
+helm.sh/chart: {{ include "onetimesecret-chart.name" . }}-{{ .Chart.Version | replace "+" "_" }}
+app.kubernetes.io/name: {{ include "onetimesecret-chart.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/version: {{ .Chart.AppVersion }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{- define "onetimesecret-chart.redis.name" -}}
+redis
+{{- end -}}
+
+{{- define "onetimesecret-chart.redis.fullname" -}}
+{{ .Release.Name }}-redis
+{{- end -}}
+
+{{- define "onetimesecret-chart.onetimesecret.name" -}}
+onetimesecret
+{{- end -}}
+
+{{- define "onetimesecret-chart.onetimesecret.fullname" -}}
+{{ .Release.Name }}-onetimesecret
+{{- end -}}
+
+{{- define "onetimesecret-service.port" -}}
+{{- if .Values.onetimesecret.service.port }}
+{{- .Values.onetimesecret.service.port -}}
+{{- else }}
+{{- 3000 -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "onetimesecret-chart.image.tag" -}}
+{{- if.Values.tag }}
+{{ .Values.tag }}
+{{- else }}
+{{ .Chart.AppVersion }}
+{{- end -}}
```

Comment:
```suggestion
{{ default .Chart.AppVersion .Values.tag }}
```
FYI you can use this syntax to simplify (this applies to other helper functions above too).
BTW note you miss a space line 47.

Reply:
Thanks for suggesting simplified way. I have applied it where it is possible in helper.

Reply:
Thank you for suggesting simplified way for defaults. I have applied default for functions where it is possible.